fuse/data/ops/ops_debug.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any, Hashable, List, Sequence, Optional
+from fuse.data.utils.sample import get_sample_id
+from fuse.utils import NDict
+from fuse.data import OpBase
+import numpy
+import torch
+
+
+class OpDebugBase(OpBase):
+    """Base class for debug operations"""
+
+    def __init__(self, sample_ids: Optional[List[Hashable]] = None, first_sample_only: bool = False):
+        """
+        :param sample_ids: apply for the specified sample ids. To apply for all set to None.
+        :param first_sample_only: apply for the first sample only
+        """
+        super().__init__()
+        self._sample_ids = sample_ids
+        self._first_sample_only = first_sample_only
+        self._first_sample_done = False
+
+    def should_debug_sample(self, sample_dict: NDict) -> bool:
+        if self._first_sample_only and self._first_sample_done:
+            return False
+
+        if self._sample_ids is not None:
+            sid = get_sample_id(sample_dict)
+            if sid not in self._sample_ids:
+                return False
+
+        self._first_sample_done = True
+        return True
+
+
+class OpPrintKeys(OpDebugBase):
+    """
+    Print list of available keys at a given point in the data pipeline
+    It's recommended, but not a must, to run it in a single process.
+    Add at the top your script to force single process:
+    ```
+    from fuse.utils.utils_debug import FuseDebug
+    FuseDebug("debug")
+    ```
+
+    Example:
+    ```
+    (OpPrintKeys(first_sample_only), dict()),
+    ```
+    """
+
+    def __call__(self, sample_dict: NDict) -> Any:
+        if self.should_debug_sample(sample_dict):
+            print(f"Sample {get_sample_id(sample_dict)} keys:")
+            for key in sample_dict.keypaths():
+                print(f"{key}")
```

Comment:
Suggestions:
1. Use `print(f"{key}", end = " ")`can be better in case there are a lot of keys. Not sure how it is common, but if there are 20+ keys printing each key in new line could cause a mess. I think thats the case int he MRI right now.. what do you think?
2. Also we can print the total number of keys.

Reply:
I think that newline for every key is easier to follow (they are also kind of sorted - so it is easier to find the key).